sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
```

Comment:
<!--thread_id:cc_158152742_t; commit:17604fb4ecfba607daa3c22312c02865b02ecee7; resolved:0-->
<!--section:context-quote-->
> **aaltay** wrote:
> Perhaps also have a JIRA issue, so that this can be fixed in other sdks.

<!--section:body-->
I am not sure if it affects other SDKs, but I have now filed BEAM-3382 for sdk-py-core